lib/internal/Magento/Framework/ObjectManager/Code/Generator/Repository.php

```diff
@@ -228,7 +228,6 @@ protected function _getGetMethod()
             'parameters' => [
                 [
                     'name' => 'id',
-                    'type' => 'int',
```

Comment:
Fair enough - I'm not really interested in working on that so should I just close this PR? I wanted the dependencies updated as we are experiencing many clashes with other packages we want to use and develop with due to these old and tight constraints.